applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -916,14 +916,8 @@ - (void)sendErrorEventWithName:(NSString *)name
                      withError:(MAError *)error
                        channel:(FlutterMethodChannel *)channel
 {
-    NSDictionary *body = ( error ) ?
-    @{@"adUnitId": adUnitIdentifier,
-      @"errorCode" : @(error.code),
-      @"errorMessage" : error.message,
-      @"waterfall": [self createAdWaterfallInfo: error.waterfall]}
-    :
-    @{@"adUnitId": adUnitIdentifier,
-      @"errorCode" : @(MAErrorCodeUnspecified)};
+    NSMutableDictionary *body = [@{@"adUnitId" : adUnitIdentifier} mutableCopy];
+    [body addEntriesFromDictionary: [self errorInfo: error]];
```

Comment:
I would try to keep parity with React Native:

https://github.com/AppLovin/AppLovin-MAX-React-Native/blob/4ec90a6f9875dcef880bd6d1cead0c52492e199f/ios/AppLovinMAX.m#L1343